pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name = "perceval"
+version = "0.17.1"
+description = "Fetch data from software repositories"
+authors = [
+    "GrimoireLab Developers"
+]
+license = "GPL-3.0+"
+
+readme = "README.md"
+
+homepage = "https://chaoss.github.io/grimoirelab/"
+repository = "https://github.com/chaoss/grimoirelab-perceval"
+
+keywords = [
+    "development",
+    "grimoirelab"
+]
+
+packages = [
+    { include = "perceval" },
+    { include = "tests", format = "sdist" },
+]
+
+classifiers = [
+   "Development Status :: 4 - Beta",
+   "Intended Audience :: Developers",
+   "Intended Audience :: Science/Research",
+   "Topic :: Software Development",
+   "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+   "Programming Language :: Python :: 3"
+]
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/chaoss/grimoirelab-perceval/issues"
+
+[tool.poetry.scripts]
+perceval = 'bin.perceval:main'
```

Comment:
To support this feature, we need to change the perceval script from `bin/perceval` to `bin/perceval.py`. If we don't need this change, we can remove these two lines.

Reply:
I'd like to keep perceval script as it is. No `.py` extensions.

Reply:
Okay, I'm removing the `[tool.poetry.scripts]` as it doesn't support in this case. :+1: